terraform/deployments/vpc/variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "govuk_environment" {
+  type        = string
+  description = "GOV.UK environment where resources are being deployed"
+}
+
+variable "aws_region" {
+  type        = string
+  description = "AWS region"
+  default     = "eu-west-1"
+}
+
+variable "stackname" {
+  type        = string
+  description = "Stackname"
+  default     = ""
+}
```

Comment:
optional: we might be able to drop "stackname" already (that's the legacy "blue" name label that appears in some domain names) — no worries if not though.

Reply:
It wasn't in use anyway, I just forgot to remove it from the terraform that was copied from the old repo